gemet/thesaurus/templates/concept_edit.html

```diff
@@ -65,20 +65,20 @@ <h5 class="h5 hidden-desktop">Definition</h5>
   <!--Source-->
   <p>
     <span>Source:</span>
-    {% include 'edit/property.html' with property_name='source' html_tag='textarea' %}
+    {% include 'edit/property.html' with property_name='source' html_tag='input' edit_concept=concept %}
     {% if not concept.source %}
       <p class="alert" id="sourceEmpty"> Source is not available. </p>
     {% endif %}
   </p>
 
    <!-- Themes -->
   <h5 class="h5">Themes</h5>
-  {% include 'edit/relation_buttons.html' with type='theme' button_label='Add Theme' %}
-  {% include 'edit/theme_group.html' with type='theme' concept_list=concept.themes %}
+  {% include 'edit/relation_buttons.html' with type='theme' button_label='Add Theme' edit_concept=concept link='false' %}
+  {% include 'edit/theme_group.html' with type='theme' concept_list=concept.themes edit_concept=concept %}
```

Comment:
I don't like the fact that you are sending `edit_concept` as a parameter to the html snippet. You could have done one of two things:

* send the `concept` parameter for those templates where the object has a different name (e.g. `concept=group`);
* change `context_object_name` for the edit views that have a name different than `concept`;